ChangeLog

```diff
@@ -1,3 +1,12 @@
+1.2.9 released
```

Comment:
Possibly missing references to (unless you consider them to be too minor to mention in the changelog):

https://github.com/arvidn/libtorrent/pull/4968
https://github.com/arvidn/libtorrent/pull/5019
https://github.com/arvidn/libtorrent/pull/5020